moped-editor/src/views/projects/projectView/ProjectComponents/ComponentListItem.js

```diff
@@ -122,7 +122,7 @@ export default function ComponentListItem({
               <ListItem className={classes.nested}>
                 <Grid container spacing={0.5}>
                   {component.moped_proj_component_work_types.map((element) => (
-                    <Grid item key={element.id}>
+                    <Grid item key={element.moped_work_type.id}>
```

Comment:
i forgot where the ProjectPhaseForm was and thought it was on the map, saw a warning about this grid item not having a key so i fixed it. Then i realized i was in the wrong place!